sim/firesim-lib/src/main/scala/util/Configs.scala

```diff
@@ -53,5 +58,17 @@ class  F65MHz extends WithDesiredHostFrequency(65)
 class  F60MHz extends WithDesiredHostFrequency(60)
 class  F55MHz extends WithDesiredHostFrequency(50)
 class  F50MHz extends WithDesiredHostFrequency(50)
+class  F45MHz extends WithDesiredHostFrequency(45)
+class  F40MHz extends WithDesiredHostFrequency(40)
+class  F35MHz extends WithDesiredHostFrequency(35)
+class  F30MHz extends WithDesiredHostFrequency(30)
```

Comment:
lol, sneaking in some lower Host Frequencies in this PR?

Reply:
Increasing window size necessitates setting a lower target frequency, I've had to go below 50 MHz for large boom configs with L2, asserts, and deep ILA.

Reply:
ok